adagrams/game.py

```diff
@@ -1,11 +1,109 @@
+import random
+
+LETTER_DISTRIBUTION = {
+    "A": 9,
+    "B": 2,
+    "C": 2,
+    "D": 4,
+    "E": 12,
+    "F": 2,
+    "G": 3,
+    "H": 2,
+    "I": 9,
+    "J": 1,
+    "K": 1,
+    "L": 4,
+    "M": 2,
+    "N": 6,
+    "O": 8,
+    "P": 2,
+    "Q": 1,
+    "R": 6,
+    "S": 4,
+    "T": 6,
+    "U": 4,
+    "V": 2,
+    "W": 2,
+    "X": 1,
+    "Y": 2,
+    "Z": 1
+}
+
+SCORE_CHART = {
+    1: ["A", "E", "I", "O", "U", "L", "N", "R", "S", "T"],  
+    2: ["D", "G"],
+    3: ["B", "C", "M", "P"],
+    4: ["F", "H", "V", "W", "Y"],
+    5: ["K"],
+    8: ["J", "X"],
+    10: ["Q", "Z"]
+}
+
 def draw_letters():
-    pass
+    letter_pool = []
+    letter_bank = []
+    for letter, count in LETTER_DISTRIBUTION.items():
```

Comment:
This works perfectly well! I have a couple of suggestions - you don't really need to turn the `LETTER_DISTRIBUTION` dictionary in to a list of tuples in order to access the count nor do you need to use a for-loop with an unused variable `i`. Here's an alternative based on your approach which doesn't convert the dictionary to a list of tuples or create unused variables:
```python
    for letter in LETTER_DISTRIBUTION:
        count = LETTER_DISTRIBUTION[letter]
        while count > 0:
            letter_pool.append(letter)
            count -= 1
```